src/DIRAC/DataManagementSystem/Client/test/Test_FTS3Objects.py

```diff
@@ -181,6 +172,20 @@ def monkeypatchForAllTest(monkeypatch):
         lambda _self, _seName, _vo: S_OK(),
     )
 
+    def mock_init(self, useProxy=False, vo=None):
```

Comment:
Why is that necessary here and below now ?

Reply:
Before tests were never going through RSS for finding the status of a SE, now they do. So, some mocking of `StorageFactory` had to be added.